pkg/selinux/setroubleshoot-view.jsx

```diff
@@ -254,11 +280,22 @@ var SETroubleshootPage = React.createClass({
             }
             var entries = this.props.entries.map(function(itm) {
                 itm.runFix = self.props.runFix;
+                var listingCaption;
+                if (itm.details && 'firstSeen' in itm.details) {
+                    if (itm.details.reportCount >= 2)
+                        listingCaption = cockpit.format(_("Occurred between $0 and $1"), itm.details.firstSeen, itm.details.lastSeen);
+                    else
+                        listingCaption = cockpit.format(_("Occurred $0"), itm.details.firstSeen);
```

Comment:
Is it intentional to display ISO 8601 formatted dates here? They're not very easy to read...